plugins/table.md

```diff
@@ -455,3 +427,9 @@ tinymce.activeEditor.plugins.table.insertTable(2, 3);
 | newcell | Fired when a cell is created |
 | ObectResizeStart | Fired when a resize action is started on a table, row, column or cell using the resize bars |
 | ObjectResized | Fired when a resize action is finished on a table, row, column or cell using the resize bars |
+
+## Commands
```

Comment:
The `nav.yml` will need updating to reflect this is now below the API instead of above it.

As a side note, should we be adding the commands to the nav for all plugins?

Reply:
wow, the plugins nav is a mess..... next build system needs to have a decent, leveled nav.

Reply:
resolving this issue for now, I'll revise the nav properly later

Reply:
Did you at least fix up the tables nav? as that'd be confusing/broken with the changes in this PR.

Reply:
Fixed... sort of.